Docs/source/running_cpp/parameters.rst

```diff
@@ -1216,10 +1216,21 @@ Diagnostics and output
 * ``<reduced_diags_name>.extension`` (`string`) optional (default `txt`)
     The extension of the output file.
 
-* ``<reduced_diags_name>.separator`` (`string`) optional (default `,`)
+* ``<reduced_diags_name>.separator`` (`string`) optional (default a `whitespace`)
     The separator between row values in the output file.
-    The default separator is comma, i.e. the output file is in
-    the CSV (comma separated value) format.
+    The default separator is a whitespace.
+    If one uses a python script to read the data,
+    one example is the following:
+
+    ::
```

Comment:
```suggestion
    ..code-block:: python
```

Reply:
This will definitely work right? It didn't work on my local machine using `restview`.

Reply:
```
.. code-block:: python
```
sorryy, it's probably because it should not be indented.